packages/react-slider/src/components/Slider/useSlider.ts

```diff
@@ -69,8 +69,8 @@ export const useSlider = (props: SliderProps, ref: React.Ref<HTMLInputElement>):
         ref,
         ...nativeProps.primary,
         type: 'range',
-        orient: vertical ? 'vertical' : '',
-      } as SliderSlots['input'],
+        orient: vertical ? 'vertical' : 'horizontal',
```

Comment:
Perhaps leave it as undefined for the horizontal case, so at least we're not setting a non-standard prop for horizontal sliders.
```suggestion
        orient: vertical ? 'vertical' : undefined,
```